File: src/utils/apiConfig.dart
class ApiConfig {
  static String get baseUrl {
    // Default to the production backend for all platforms to match the website.
    // Use 'http://10.0.2.2:5000' for Android Emulator local testing if needed.
    return 'https://scanserve.in';
    // return 'https://zpjn3sn1-5000.inc1.devtunnels.ms';
  }

  // If moving to production later, this should be swapped:
  // static const String baseUrl = 'https://restaurant-model-backend.onrender.com';
}

File: src/utils/apiConfig.js
//const apiConfig = {
 // BASE_URL: process.env.NEXT_PUBLIC_API_URL || "http://localhost:5000",
//};

//export default apiConfig;
//

const BASE_URL =
  process.env.NEXT_PUBLIC_API_URL ||
  "https://scanserve.in" 
  // "http://localhost:5000";

const apiConfig = {
  BASE_URL,
};

export default apiConfig;
